app/femr/util/calculations/dateUtils.java

```diff
@@ -42,7 +43,7 @@ public static String getCurrentDateTimeString(){
     }
 
     public static DateTime getCurrentDateTime(){
-        return new DateTime();
+        return new DateTime(DateTimeZone.UTC);
```

Comment:
Sam -

Having trouble identifying the significance of this change along with line 100 after changing the JDBC connection string, is there one?

-Kevin